<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>861</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: 'white'
}
</string>
   </property>
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>0</y>
      <width>891</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>truck2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text1Left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: 'white';
font: 75 48pt &quot;League Spartan&quot;;
</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2Left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>700</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: 'white';
font: 25 15pt &quot;Calibri&quot;;
</string>
    </property>
    <property name="text">
     <string>Enter your credentials to login</string>
    </property>
   </widget>
   <widget class="QLabel" name="text3Left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: 'white';
font: 25 15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Forgot password? Contact the head manager for a reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="TCCS_heading">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 48pt &quot;League Spartan&quot;;
color: rgb(222, 24, 24)</string>
    </property>
    <property name="text">
     <string>TCCS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernamelineEdit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>360</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff3f3;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: #fff3f3;
padding: 4px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordlineEdit">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>450</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff3f3;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: #fff3f3;
padding: 4px</string>
    </property>
   </widget>
   <widget class="QLabel" name="text1_right">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>220</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Enter username and password</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2_right">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>240</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Do your job or you will be fired:) No pressure&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>540</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;League Spartan&quot;;
color: white;
background-color: rgb(222, 24, 24);
border-radius: 15px</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_text">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>330</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 24, 24);
font: 75 15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_text">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 24, 24);
font: 75 15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
